docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -0,0 +1,77 @@
+# Why Toolpad?
+
+<p class="description">Learn how Toolpad approaches building internal tools, and how it can be useful to you.</p>
+
+## TL;DR
+
+- **Higher-level API**. Toolpad operates at a higher abstraction level than Material UI. It sets healthy UI constraints that prevent common pitfalls and anti-patterns
+- **Drag-and-drop builder**. Toolpad comes with a drag-and-drop builder to simplify UI building. The Drag-and-drop state is persisted in a yaml file so you can fully control the visual output with code
+- **Less boilerplate** so you can focus on the important parts of the app
+- **Run alongside existing code**. It integrates well with your existing code. You can use your database models, client libraries, secrets, and bespoke components directly
+- **Single source of truth**. Because Toolpad runs alongside your existing code, it lets you maintain a single source of truth for all business logic
+- **Integrates well with your development lifecycle**. Your project lives on your own file system. You can use your own IDE, version control system, deployment target, CLI tools
+
+## The problems
+
+After reviewing multiple approaches and tools, we noticed that there are a few recurring core problem areas that needed to be addressed:
```

Comment:
Based on our style guide, we should avoid using "we": "It’s better to use the passive voice when it allows you to not use “we”"